### PR TITLE
`replace_bytecode` appendix revamp

### PR DESCRIPTION
**Pull request by Arcticae**
## Introduced changes

- Deleted example, moved param explanation to cairo

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/appendix/cheatcodes/replace_bytecode.md [hunk 1]
```diff
@@ -2,65 +2,4 @@
...
-fn test_replace_bytecode() {
-    // ...
-
-    dispatcher.set(50);
-    replace_bytecode(contract_address, new_class_hash);
-
-    assert(dispatcher.get() == 150, 'wrong value');
-}
-```
+Replaces class for given contract address.
```
Comment (piotmag769) [inline docs/src/appendix/cheatcodes/replace_bytecode.md:5]: Maybe it's dumb but is it worth mentioning that it should be a class hash of an already declared contract? The cheatcode would work anyways, but then weird stuff will happen

Comment (Arcticae) [reply docs/src/appendix/cheatcodes/replace_bytecode.md:5]: Yeah that's worth mentioning 👍 
BTW: Perhaps we should safeguard this case somehow then? 🤔

Comment (Arcticae) [reply docs/src/appendix/cheatcodes/replace_bytecode.md:5]: [Add elaboration in the docs about the existance of the replaced class](https://github.com/foundry-rs/starknet-foundry/pull/2055/commits/6a68ba3d7ca26ff471d7af02a48e6953edc277dc)

File: docs/src/appendix/cheatcodes/replace_bytecode.md [hunk 2]
```diff
@@ -2,65 +2,5 @@
...
+The `new_class` hash has to be declared (in order for the replacement class to execute the code).
```
Comment (piotmag769) [inline docs/src/appendix/cheatcodes/replace_bytecode.md:6]:
```suggestion
The `new_class` hash has to be declared in order for the replacement class to execute the code when interacting with the contract.
```

**Review by piotmag769 — APPROVED** [review]

**Pull request merged**